refarch-frontend/src/api/HealthService.ts

```diff
@@ -1,10 +1,9 @@
 import FetchUtils from "@/api/FetchUtils";
-import { API_BASE } from "@/Constants";
 import HealthState from "@/types/HealthState";
 
 export default class HealthService {
   static checkHealth(): Promise<HealthState> {
-    return fetch(`${API_BASE}/actuator/health`, FetchUtils.getGETConfig())
+    return fetch(`actuator/health`, FetchUtils.getGETConfig())
```

Comment:
```suggestion
    return fetch(`/actuator/health`, FetchUtils.getGETConfig())
```
Looks cleaner in my opinion

Reply:
Absolute paths don't work here as the request will be sent to http://actuator/health then. ;) Relative path is exactly what we need because the proxy will handle the rest. ;)